packages/cli/src/log.ts

```diff
@@ -33,9 +35,9 @@ export function getRecordingSink(): RecordingSink {
 
 export const recordingSink = getRecordingSink();
 
-export const logFile = Deno.env.get("FEDIFY_LOG_FILE");
+export const logFile = process.env["FEDIFY_LOG_FILE"];
 if (logFile != null) {
-  await Deno.mkdir(dirname(logFile), { recursive: true });
+  mkdir(dirname(logFile), { recursive: true });
```

Comment:
Should this be awaited?

```suggestion
  await mkdir(dirname(logFile), { recursive: true });
```